frontend/packages/kubevirt-plugin/src/components/vm-nics/vm-nics.tsx

```diff
@@ -124,10 +127,11 @@ export const VMNics: React.FC<VMLikeEntityTabProps> = ({ obj: vmLikeEntity }) =>
                   nicModalEnhanced({
                     blocking: true,
                     vmLikeEntity,
+                    isVMRunning,
                   }).result,
                 )
               }
-              isDisabled={isLocked || isVMRunningOrExpectedRunning(asVM(vmLikeEntity))}
+              isDisabled={isLocked}
```

Comment:
we should allow editing for the nics/disks as well if we allow creating them

Reply:
@suomiy  we cannot allow editing / deleting disks and nics __at the moment__, since it could result unexpected behavior.
For example: adding a disk -> add it to the boot order -> delete the disk. result an invalid boot order.
I believe there are other edge cases like this.

IMO, we should have this functionality, but it needs to be checked and tested and not added at the last moment. I would like to avoid making the PR unstable.

Reply:
it should be the same and we should be detecting all relevant changes since it goes all through yaml. Anyway I don't mind adding this later